frontend/src/features/admin/agencies/ManageAgencies.scss

```diff
@@ -0,0 +1,37 @@
+$header-height: 72px;
+$footer-height: 46px;
+$navbar-height: 45px;
+$filter-height: 76px;
+$contentheading-height: 60px;
+.ManageAgencies {
+  background-color: white;
+  display: flex;
+  flex-direction: column;
+  padding: 0;
+  .agency-toolbar {
+    padding: 0;
+    background-color: var(--light);
+    .search-bar {
+      .filterBarHeading {
+        margin-left: 20%;
```

Comment:
% based margins can be troublesome for responsive designs, how does this page perform when it is resized?

Reply:
Terrible at the moment, all the styling is just a preliminary thing right now. Wanted to get the functionality code looked at then was going to clean this up afterwards.